composer.json

```diff
@@ -28,7 +28,7 @@
     },
     "require-dev": {
         "raulfraile/ladybug": "~1.0.8",
-        "phpspec/prophecy": "~1.2.1"
+        "phpspec/prophecy": "~1.3.1"
```

Comment:
so this was the problem, hmm yet it still fails in hhvm for some reason. Sorry i failed to see this from the beginning.